net/quagga/src/opnsense/mvc/app/controllers/OPNsense/Quagga/Api/Ospf6settingsController.php

```diff
@@ -0,0 +1,195 @@
+<?php
+namespace OPNsense\Quagga\Api;
+use \OPNsense\Base\ApiControllerBase;
```

Comment:
we can remove this line as well - the quagga plugin needs some clean up. This is left from quagga plugin 0.0.0 :)